docs/howdoi_advanced_usage.md

```diff
@@ -0,0 +1,56 @@
+- **Howdoi stashing feature** - We agree that sometimes you need to need search results for later and running the same query again and again
+  won’t be that feasible. Hence, Howdoi has a stashing feature which allows you to save your query, view the query, delete the saved
+  results and even empty the entire stash ! (see keep documentation for more information on stashing). Here is how you can do this:
+
+  - **stashing: howdoi --save QUERY** **viewing: howdoi --view**
+  - **removing: howdoi --remove (will be prompted which answer to delete)**
+  - **emptying: howdoi --empty (empties entire stash, will be prompted to confirm)**
+
+- **Shortcuts for your parameters** - You might run the same parameters many times and again, typing them isn’t always the best option. You can use shortcuts for your parameters by using something like:
+
+  ```
+  $ alias h='function hdi(){ howdoi $* -c -n 5; }; hdi'
+  ```
+
+  And the in your command line, replace your parameters with your alias i.e. h:
+
+  ```
+  $ h format date bash
+  ```
+
+- **Other uses and aliases** - You can also search other StackExchange properties for answers.
+
+  Example:
+
+  ```
+  $ HOWDOI_URL=cooking.stackexchange.com
```

Comment:
I think you can put these two lines together into a single block quote here and also `hcook` below.